src/pages/App.tsx

```diff
@@ -51,31 +52,34 @@ export default function App(): ReactElement {
         <GasAndTokenPrices>
           <PendingSwapsProvider>
             <RewardsBalancesProvider>
-              <Switch>
-                <Route exact path="/" component={Swap} />
-                <Route exact path="/pools" component={Pools} />
-                {pools.map(({ name, route }) => (
-                  <Route
-                    exact
-                    path={`/pools/${route}/deposit`}
-                    render={(props) => <Deposit {...props} poolName={name} />}
-                    key={`${name}-deposit`}
-                  />
-                ))}
-                {pools.map(({ name, route }) => (
-                  <Route
-                    exact
-                    path={`/pools/${route}/withdraw`}
-                    render={(props) => <Withdraw {...props} poolName={name} />}
-                    key={`${name}-withdraw`}
-                  />
-                ))}
-                <Redirect from="/pools/:route/:action" to="/pools" />
-                <Route exact path="/risk" component={Risk} />
-                <Route exact path="/mui" component={MuiSwap} />
-              </Switch>
-
-              <Version />
+              <Container>
```

Comment:
What is Container here for? Is there another file outside of App this can be used?